### PR TITLE
?? Add support for optional chaining ??

### PR DESCRIPTION
**Pull request by ottah**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -85,6 +84,7 @@
     "@types/node-fetch": "^2.3.7",
     "@types/openfin": "^43.0.1",
     "@types/webpack-dev-middleware": "^2.0.3",
+    "eslint": "^6.8.0",
```
Comment (pjbroadbent) [inline package.json]: How come this is now a dev dependency? Is this not needed within each of the downstream projects for the linter to run?

Comment (ottah) [reply package.json]: Oops left the `-D` in when upgrading.

**Review by ngunday — APPROVED** [review]
LGTM once you fix dev dependency bit.

File: package.json [hunk 2]
```diff
@@ -31,15 +31,14 @@
     "@types/jest": "^24.0.15",
     "@types/rimraf": "^2.0.2",
     "@types/webpack": "^4.4.35",
-    "@typescript-eslint/eslint-plugin": "^2.6.0",
-    "@typescript-eslint/parser": "^2.6.0",
+    "@typescript-eslint/eslint-plugin": "2.22.0",
```
Comment (rypete) [inline package.json:34]: How come we are locking to this version?

Comment (ottah) [reply package.json:34]: Locked the versions so we don't get the same problems we had with the import order linting.

**Comment by finos-admin** [comment]
Thank you for your pull request and welcome to our community! We require contributors to sign a [Contributor License Agreement](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530375/Legal+Requirements#LegalRequirements-ContributorLicenseAgreement) and we don't seem to have CLAs on file for these contributors to the Pull Request: (@ottah). In order for your PR to be reviewed and merged, please follow the directions at the link above.

Project team: please **do not merge this Pull Request** until Foundation staff have confirmed that a CLA is in place for the new contributor(s) listed above.

If there are any questions, please don't hesitate to [get in touch with our Infrastructure Support team](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/77955298/Getting+in+Touch#GettinginTouch-InfrastructureSupport). /CC @finos-admin

**Comment by finos-admin** [comment]
Thank you for your pull request and welcome to our community! We require contributors to sign a [Contributor License Agreement](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530375/Legal+Requirements#LegalRequirements-ContributorLicenseAgreement) and we don't seem to have CLAs on file for these contributors to the Pull Request: (@ottah). In order for your PR to be reviewed and merged, please follow the directions at the link above.

Project team: please **do not merge this Pull Request** until Foundation staff have confirmed that a CLA is in place for the new contributor(s) listed above.

If there are any questions, please don't hesitate to [get in touch with our Infrastructure Support team](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/77955298/Getting+in+Touch#GettinginTouch-InfrastructureSupport). /CC @finos-admin

**Comment by ottah** [comment]
@finos-cla-bot[bot] check

**Comment by finos-admin** [comment]
The cla-bot has been summoned, and re-checked this pull request!

**Review by pjbroadbent — APPROVED** [review]

**Pull request merged**